cia.mk

```
@@ -1,59 +1,91 @@
 ### Virtual Console repacking stuff
 
-# Include Configuration Settings
 vc_dir := poke-cia
--include $(vc_dir)/cia-config.mk
+
+# Include Configuration Settings
+include ${vc_dir}/cia-config.mk
+ifeq ($(strip ${vc_name}),)
+$(error Please set the `vc_name` variable in ${vc_dir}/cia-config.mk)
+endif
+
+vc_rom_dirs   := $(addprefix ${vc_dir}/, ${vc_name})
+vc_cias       := $(addsuffix .cia, ${vc_rom_dirs})
+vc_orig_cia   := $(addsuffix .orig.cia, ${vc_rom_dirs})
+vc_game_cxi   := $(addsuffix .game.cxi, ${vc_rom_dirs})
+vc_manual_cfa := $(addsuffix .manual.cfa, ${vc_rom_dirs})
+
```

This should probably be included to specify the names of the requirements of cxi

```suggestion
vc_cxi_req    := exheader.bin logo.lz plain.bin exefs/banner.bin exefs/code.bin exefs/icon.bin romfs/

```

---

Yeah, it'll be part of getting rid of the directory dependency if we go with the second option I suggested above—so I'll take this as your agreement that we should go with that ;3